distributed/system_monitor.py

```diff
@@ -142,7 +141,7 @@ def update(self) -> dict[str, Any]:
             self._last_net_io_counters = net_ioc
 
         if self.monitor_disk_io:
-            disk_ioc = psutil.disk_io_counters()
+            disk_ioc = cast(psutil._common.sdiskio, psutil.disk_io_counters())
```

Comment:
Perhaps
```suggestion
            disk_ioc = psutil.disk_io_counters()
            assert disk_ioc is not None
```

Is a nicer approach.

Reply:
Yes I prefer it too